runtime/Jakt/StringBuilder.h

```diff
@@ -49,7 +49,7 @@ class StringBuilder {
     [[nodiscard]] bool is_empty() const { return m_buffer.is_empty(); }
 
     template<class SeparatorType, class CollectionType>
-    ErrorOr<void> join(SeparatorType const& separator, CollectionType const& collection, StringView fmtstr = "{}"sv)
+    void join(SeparatorType const& separator, CollectionType const& collection, StringView fmtstr = "{}"sv)
```

Comment:
This looks wrong. Now ErrorOrs from append and appendff will be dropped on the floor -- which shouldn't be possible, ErrorOr is [[nodiscard]]

Reply:
Yeah I think this is the wrong fix looking now, it probably just needs a return at the end, I think `return {};`